rke/templates/flannel_v0.14.0-rancher2.go

```diff
@@ -0,0 +1,284 @@
+package templates
+
+/*
+FlannelTemplateV0_14_0Rancher2 is based on upstream flannel v0.14.0
+Source: https://raw.githubusercontent.com/flannel-io/flannel/v0.14.0/Documentation/kube-flannel.yml
+
+Rancher Changelog:
+- Remove resource limits for kube-flannel DaemonSet
```

Comment:
Let's reference upstream PR here so it doesn't look like Rancher-specific change - https://github.com/flannel-io/flannel/pull/1694

Reply:
done